content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: OpenGitOps 1.0 is finally here and why you should care
+date: "2021-11-03T09:00:01.284Z"
+description: "OpenGitOps 1.0 brings a mature standard to the DevOps industry."
+author: Dan Garfield and Scott Rigby
+thumbnail: images/GitOps-1.0.jpg
+---
+
+GitOps is a term coined by Alexis Richardson a few years ago and has grown immensely in popularity.
+In a nutshell, GitOps is a set of principles for operating and managing software systems, derived from modern software operations, but also rooted in pre-existing and widely adopted best practices.
+However they differ in specific ways from traditional CI/CD pipeline approach that has dominated the industry for decades.
```

Comment:
```suggestion
However they differ in specific ways from traditional CI & CD pipeline approach that has dominated the industry for decades.
```
@todaywasawesome @scottrigby @murillodigital I agree with @countspongebob to not conflate CI and CD hence my suggestion here to change "CI/CD" to "CI & CD".

From https://www.youtube.com/watch?v=o2_sGFkYNvA&t=1184s by @countspongebob
> It is really critical to not conflate CI and CD.  I try hard to not use the phrase CI/CD because they are very different practices.  One of the lovely things about the GitOps model is the clean and understandable relationship that is created between CI as a Dev practice, and CD as an Ops practice.  I think the notion of the Immutability Firewall in between them based on Git is really excellent.